invenio_records_rest/config.py

```diff
@@ -54,6 +54,19 @@ def _(x):
         item_route='/records/<pid_value>',
         default_media_type='application/json',
         max_result_window=10000,
+        suggestions=[
```

Comment:
I tried to be more generic as possible. In this case each element of the array is a suggestion.
If e.g you need to define different suggestions for different fields.